Neos.Neos/Documentation/References/NeosFusionReference.rst

```diff
@@ -364,6 +424,21 @@ Evaluate nested definitions as an array (opposed to *string* for :ref:`Neos_Fusi
 
 .. tip:: For simple cases an expression with an array literal ``${[1, 2, 3]}`` might be easier to read
 
+.. note:: The Neos.Fusion:RawArray object has been renamed to Neos.Fusion:Dictionary the old name is &lt;b&gt;(DEPRECATED)&lt;/b&gt;
```

Comment:
Is `&lt;b&gt;` intended here?

Reply:
I simply found that in another deprecation notice. Not much intention from my side other than that